### PR TITLE
Improve Test Accounts document

### PR DESCRIPTION
**Pull request by ryanlink**
2nd pass at this.  I'm hoping someone has a shortcut to removing the Password column for Health API users and moving the Usernames to the leftmost column for both sets.

### CODE REVIEW

**Comment by AubreySuter** [comment]
Made some changes to the tables along the lines of what we'd talked about. Looks like my commits somehow broke the GitHub preview, which is scary, but the .md file renders nicely through https://stackedit.io/app# so I think we should be good.

File: test_accounts.md [hunk 1]
```diff
@@ -22,163 +23,168 @@ The Representative users 001 (Tamara Ellis) and 007 (John Doe) have Power of Att
...
+|va.api.user+idme.066@gmail.com|LEONA|C|WILSON|F|1948-09-28|796066619|
+|va.api.user+idme.067@gmail.com|JAMIE|J|WOOD|F|1945-05-24|796060339|
+|va.api.user+idme.068@gmail.com|TOMMY|V|WRIGHT|M|1996-11-17|796030117|
+
 
 ## Health API Accounts
 
 These users have a variety of synthetic health records defined for use with the VA Argonaut Health APIs.
 
-| Email | Password      | Last Name | First Name | Gender | Birthdate |
```
Comment (AubreySuter) [inline test_accounts.md:100]: Not necessarily something we need to address before merging this, but are "Gender" and "Sex" used differently between Representative and Health systems? Just jumped out at me as an inconsistency between the tables and I'm not sure if there's preferred language or a reason they should be different.

**Comment by ryanlink** [comment]
LGTM, thanks for your work on this @careyotto and @AubreySuter!  

I added a reference to Veteran Verification in two spots for the non-Health users because they can be used to test that API as well.

Only one remaining question:  what do we make of the two non-Health Veterans that do not have SSNs?  (009 and 010). @bastosmichael @charleystran do you know?  I can't think of any use case where Veteran data could be pulled without an SSN, so maybe these two are also PoA reps?

image

**Comment by bastosmichael** [comment]
Claims V0/V1 requires SSN, Date of Birth, First and Last name for Representative Oauth to work and be testable by the consumer which is why we had it in here in the first place, Health API's are veteran only and don't have this feature so they don't require ssn.

**Comment by ryanlink** [comment]
@bastosmichael the two test users in question are in the non-Health category.  all other non-Health users have SSNs.

**Pull request merged**